manifests/c/com/exelearning/editor/2.7/com.exelearning.editor.locale.es-ES.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: com.exelearning.editor
+PackageVersion: 2.7
+PackageLocale: es-ES
+Publisher: eXelearning
+PackageName: eXelearning
+License: https://www.exelearning.org/en/terms-and-conditions/
+ShortDescription: eXelearning is a software used to create and publish e-learning content.

```

Comment:
Either this needs to be translated to Spanish as `es-ES` locale is used or the locale fields need to be changed to `en-US`